### PR TITLE
Dev-->Staging

### PR DESCRIPTION
**Pull request by jennmalcolm**
[:sunglasses: PREVIEW](https://federalist-proxy.app.cloud.gov/preview/onrr/doi-extractives-data/dev/)

Changes proposed in this pull request:

- Updates federal calendar year production data
- Clears curated search terms
- Styleguide fixes
- Adds core site components in Gatsby
- Updates CircleCI
- Updates packages

### CODE REVIEW

**Comment by jennmalcolm** [comment]
@master12 or @brentryanjohnson is this ok to have #3015 in here when those changes were merged to master in #3034?

**Comment by brentryanjohnson** [comment]
I think it's fine, @jennmalcolm. I'm fairly confident it's just recording the action of the merge, not the changes which were merged.

**Comment by jennmalcolm** [comment]
Ok @brentryanjohnson thanks

File: gatsby-site/src/components/Footer/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <div className="container-page-wrapper">
+    <div className="footer-col_left">
+      <a href="https://doi.gov"><img className="footer-image" src={DoiLogo} alt="Department of the Interior logo" /></a>
+    </div>
+
+    <div className="footer-col_right">
+
+      <div className="footer-bottom-left">
+	      <p className="footer-para_callout footer-para_callout-bigger">Built in the open</p>
+	      <p className="footer-para">This site (<a href={"https://github.com/18F/doi-extractives-data/releases/"+siteMetadata.version } className="link-active-beta">{ siteMetadata.version }</a>) is powered by <a className="link-active-beta" href={ siteMetadata.url+"/downloads"}>open data</a> and <a className="link-active-beta" href="https://github.com/ONRR/doi-extractives-data/">source code</a>. We welcome contributions and comments on <a className="link-active-beta" href="https://github.com/ONRR/doi-extractives-data/issues/new">GitHub</a>.</p>
```
Comment (brentryanjohnson) [inline gatsby-site/src/components/Footer/index.js]: @master12 You may want to correct the repo path in your Gatsby footer.

**Pull request merged**